test/signatures.jl

```diff
@@ -103,6 +103,7 @@ bodymethtest5(x, y=Dict(1=>2)) = 5
     modeval, modinclude = getfield(Lowering, :eval), getfield(Lowering, :include)
     failed = []
     for fsym in nms
+        isdefined(Lowering, fsym) || continue
```

Comment:
What's this change about? If we didn't need this before why do we need it now?

Reply:
Don't exactly know. One of the expressions in the list is leaking an unused gensym. Probably should be cleaned up, but in any case, not an LCU issue.

Reply:
Let me look before we merge this then.

Reply:
I find I can delete this line and the signatures test still passes.

Reply:
Let's delete it then. Maybe it was something transient.